.circleci/scripts/git-diff-develop.ts

```diff
@@ -34,8 +47,10 @@ async function getBaseRef(): Promise<string | null> {
  */
 async function fetchWithDepth(depth: number): Promise<boolean> {
   try {
-    await exec(`git fetch --depth ${depth} origin develop`);
-    await exec(`git fetch --depth ${depth} origin ${process.env.CIRCLE_BRANCH}`);
+    await exec(`git fetch --depth ${depth} origin ${MAIN_BRANCH}`);
+    await exec(
+      `git fetch --depth ${depth} origin ${SOURCE_BRANCH}:${SOURCE_BRANCH}`,
```

Comment:
```suggestion
    await exec(`git fetch --depth ${depth} origin "${MAIN_BRANCH}"`);
    await exec(
      `git fetch --depth ${depth} origin "${SOURCE_BRANCH}:${SOURCE_BRANCH}"`,
```
Might as well quote all shell args with templated input?